toolchains/xbox/CMakeGDKScarlett.cmake

```diff
@@ -0,0 +1,242 @@
+#
+# CMakeGDKScarlett.cmake : CMake definitions for Microsoft GDK targeting Xbox Series X|S
+#
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
```

Comment:
Could you also add a link to where this came from with the commit sha in the link? This helps us see diff later and pull updates, if there are some.

And also help show that it came from an open repo as opposed to anything closed source (even though it's MIT licensed)

Reply:
Added links to each of the CMake sources from public Microsoft repos.